x/wasm/genesis_test.go

```diff
@@ -30,7 +30,10 @@ func TestInitGenesis(t *testing.T) {
 
 	res, err := h(data.ctx, &msg)
 	require.NoError(t, err)
-	require.Equal(t, res.Data, []byte("1"))
+
+	var pStoreResp MsgStoreCodeResponse
```

Comment:
If we do similar tests a lot (eg. in handler tests), it may be useful to have helper methods for them

Reply:
Added